packages/eslint-plugin-svelte/src/rules/prefer-const.ts

```diff
@@ -0,0 +1,99 @@
+import { createRule } from '../utils';
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { defineWrapperListener, getCoreRule } from '../utils/eslint-core';
+
+const coreRule = getCoreRule('prefer-const');
+
+type Declaration = TSESTree.Expression;
+
+/**
+ * Finds and returns the callee of a declaration node within variable declarations or object patterns.
+ */
+function findDeclarationCallee(node: Declaration) {
```

Comment:
I don't think we need a type alias.

```suggestion
function findDeclarationCallee(node: TSESTree.Expression) {
```